pkg/clusterdeletion/volumes.go

```diff
@@ -48,10 +53,18 @@ func (d *Deletion) cleanupVolumes(ctx context.Context, cluster *kubermaticv1.Clu
 		return false, fmt.Errorf("failed to list PVCs from user cluster: %v", err)
 	}
 
-	pvList := &corev1.PersistentVolumeList{}
-	if err := userClusterClient.List(ctx, pvList); err != nil {
+	allPVList := &corev1.PersistentVolumeList{}
+	if err := userClusterClient.List(ctx, allPVList); err != nil {
 		return deletedSomeResource, fmt.Errorf("failed to list PVs from user cluster: %v", err)
```

Comment:
@wozniakjan I know it's not you that introduced this, maybe I'm missing something but what's the point of returning `deletedSomeResource` here as it is never set before?

```suggestion
		return false, fmt.Errorf("failed to list PVs from user cluster: %v", err)
```

Reply:
you are correct that for this particular `return` statement, `deletedSomeResource` is always going to be `false`. It's only the following `return` statements that may be `true` depending on `PVs` and `PVCs`. Your suggestion makes sense to me.